docs/Wallet.md

```diff
@@ -25,70 +19,80 @@ The wallet contains following features:
+[Check instructions](Vault.md) on how to set up and sign with a [compatible hardware wallet](https://github.com/bitcoin-core/HWI#device-support).
 
-#### Signing with a Hot Wallet
+##### Signing with a hot wallet
 
-BTCPay Server also allows stores to generate or import a wallet while also storing its private keys. If your store was configured this way, you'll have the option available in the "Sign with.." dropdown.
+If you [created a new wallet](CreateWallet.md) when setting up your store and enabled a [hot wallet](HotWallet.md) you can sign a transaction with a private key stored on your server.
 
-[Using the hot wallet feature comes with its own security implications, please be sure to read and understand them over at the Hot Wallet documentation.](HotWallet.md)
+:::danger
+Using the hot wallet feature comes with its own security implications, please be sure to read and understand them over at the [Hot Wallet documentation](HotWallet.md)
+:::
 
-### Receive
-
-Receive tab generates an unused address which can be used to receive payments. The same can be achieved by generating an invoice (Invoices > Create new invoice).
+#### Advanced Settings
 
-### Advanced Settings
+Certain wallet features are available for advanced users. Toggle the `Advanced Settings` within the `Send` tab to preview them.
 
 #### Don't create UTXO change
 
-This option is available in the `Advanced mode` of the Send page.
+This option is available in the `Advanced mode` of the `Send` page.
 
-It is a privacy enhancing feature which is useful when you send to another of your own wallets or to an exchange. It makes sure that no change UTXO is created by **rounding up** the amount sent.
+It is a privacy enhancing feature which is useful when you're sending funds to another wallet of yours or to an exchange. It makes sure that no change UTXO is created by **rounding up** the amount sent.
 
-By default this feature is disabled, so if your wallet has a UTXO of `1.1 BTC` and you input an amount equals to `1.0 BTC`, the resulting transaction will have two outputs `0.1 BTC` of change, and `1.0 BTC` to your destination.
+By default this feature is disabled, so if your wallet has an UTXO of `1.1 BTC` and you input an amount equals to `1.0 BTC`, the resulting transaction will have two outputs `0.1 BTC` of change, and `1.0 BTC` to your destination.
 
 Blockchain analysis will understand that those `0.1 BTC` of change belong to the same entity which controlled `1.1 BTC` before, and can track the future purchase you make under the same pattern.
 
 By enabling this feature, BTCPay Server wallet will round up the amount sent to `1.1 BTC` such that no change output is sent back to you.
 
 Warning: Despite the fact, in this example, that you entered `1.0` in the amount field, the amount that will really be sent to your destination will be `1.1 BTC`.
 
+##### RBF (Replace-by-fee)
```

Comment:
@britttttk I know you wanted to document this, if you want you can commit it here.

Reply:
There is some new settings around this in the next release. By default, it will now randomize it for more privacy.